Packs/Endace/Playbooks/playbook_Endace_Search_Archive_Download.yml

```diff
@@ -0,0 +1,606 @@
+      description:
+        simple: PCAP FIleName and Size
+      entryIDs: {}
+      occurred: {}
+      tags: {}
+    separatecontext: false
+    view: "{\n  \"position\": {\n    \"x\": 655,\n    \"y\": 2285\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  '26':
+    id: '26'
+    taskid: bc5bf4db-a94d-4409-832f-545670da5b8e
+    type: condition
+    task:
+      id: bc5bf4db-a94d-4409-832f-545670da5b8e
+      version: -1
+      name: Is Bytes Count > 0?
+      description: "Checks if one number(float) as bigger than the other(float)\n\
+        Returns yes: if first > second\nReturns no: if first <= second\nReturns exception\
+        \ if one of the inputs is not a number"
+      scriptName: IsGreaterThan
+      type: condition
+      iscommand: false
+      brand: ''
+    nexttasks:
+      no:
+      - '7'
+      yes:
+      - '36'
+    scriptarguments:
+      first:
+        complex:
+          root: Endace
+          filters:
+          - - operator: greaterThan
+              left:
+                value:
+                  simple: Endace.Search.Response.TotalBytes
+                iscontext: true
+              right:
+                value:
+                  simple: '0'
+          accessor: Search.Response.TotalBytes
+      second:
+        simple: '0'
+    separatecontext: false
+    view: "{\n  \"position\": {\n    \"x\": 270,\n    \"y\": 890\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  '27':
+    id: '27'
+    taskid: ea63c7a0-3fe2-4d44-8775-46212d2199a5
+    type: condition
+    task:
+      id: ea63c7a0-3fe2-4d44-8775-46212d2199a5
+      version: -1
+      name: Is Archived Finished ?
+      description: Check whether the values provided in arguments are equal. If either
+        of the arguments are missing, no is returned.
+      scriptName: AreValuesEqual
+      type: condition
+      iscommand: false
+      brand: ''
+    nexttasks:
+      no:
+      - '7'
+      yes:
+      - '14'
+    scriptarguments:
+      left:
+        simple: ${Endace.Archive.Response.Status}
+      right:
+        simple: Finished
+    results:
+    - AreValuesEqual
+    separatecontext: false
+    view: "{\n  \"position\": {\n    \"x\": 275,\n    \"y\": 1770\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  '28':
+    id: '28'
+    taskid: a07ae568-53dc-455f-839e-a4f9ffae4d5b
+    type: regular
+    task:
+      id: a07ae568-53dc-455f-839e-a4f9ffae4d5b
+      version: -1
+      name: endace-create-search
+      description: Create a Search Job on EndaceProbe. Search is issued against active
+        and archived datasources on all EndaceProbes
+      script: Endace|||endace-create-search
+      type: regular
+      iscommand: true
+      brand: Endace
+    nexttasks:
+      '#none#':
+      - '29'
```

Comment:
Generic polling should be used at this point and not in task 21

Reply:
looks like task ID in playbook file is completely out of order from the visual look and execution of the playbook.